assignments/prototypes.js

```diff
@@ -32,6 +52,22 @@
   * should inherit destroy() from GameObject through CharacterStats
   * should inherit takeDamage() from CharacterStats
 */
+
+function Humanoid(myObj) {
+  CharacterStats.call(this, myObj)
+  this.faction = myObj.faction
+  this.weapons = myObj.weapons
+  this.language = myObj.language
+}
+
+Humanoid.prototype = Object.create(GameObject.prototype)
+Humanoid.prototype = Object.create(CharacterStats.prototype)
```

Comment:
Passing characterStats as a second parameter might also help redundancy. The first declaration on Line: 63, is being immediately overwritten on the next line. However you should only need to pass the parent.